listener/src/event_handler/util.rs

```diff
@@ -11,10 +12,88 @@ pub fn entity_id_from_log(log: &Arc<Log>) -> String {
     )
 }
 
-pub fn entity_key(entity_type: &str, entity_id: &str) -> EntityKey {
+pub fn entity_key(entity_type: &str, entity: &Entity) -> EntityKey {
     EntityKey {
         subgraph_id: SUBGRAPH_ID.clone(),
         entity_type: entity_type.to_string(),
-        entity_id: entity_id.to_string(),
+        entity_id: entity.get("id")
+            .and_then(|v| v.clone().as_string())
+            .unwrap(),
+    }
+}
+
+pub fn entity_query(entity_type: &str, filter: EntityFilter) -> EntityQuery {
+    EntityQuery {
+        subgraph_id: SUBGRAPH_ID.clone(),
+        entity_types: vec![entity_type.to_string()],
+        filter: Some(filter),
+        order_by: None,
+        order_direction: None,
+        range: EntityRange {
+            first: None,
+            skip: 0
+        }
+    }
+}
+
+#[cfg(test)]
+pub mod test {
```

Comment:
I don't see any assertions in these tests? Is this normal?

Reply:
It doesn't test anything but just provides test data for other threads.